demo/App.js

```diff
@@ -14,12 +17,20 @@ const customClassNames = {
   'rq-Quiz-nextButton': 'btn btn-primary pull-right'
 }
 
+// You can override text in a similar way, e.g. if you want to translate the
+// component to different languages.
+const customText = {
+  // 'rq-Quiz-progressText': 'Question {n} of {total}',
```

Comment:
All of the comments in this file were intended as documentation to show how you can integrate React Quiz into a React project and customise it. But we can definitely change things if it'll make it clearer.